department-of-veterans-affairs/va.gov-cms
# VACMS-13467: Updated notification for vet center outdated content

## Description
Closes #13467.

## Testing done
Tested locally, Ran drush command to generate queue and then ran cron manually to send emails. Inspected emails caught by mailhog. 

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

Engineer
1. Shell into tugboat and run `drush php-eval "print_r(\Drupal::service('va_gov_notifications.outdated_content')->checkForOutdatedVetCenterContent());"`
   - [x] Validate that messages are queued by visiting `/admin/config/system/queues`
2. Then send notification by running cron at `/admin/config/system/cron`
   - [x] Validate that notifications are sent be checking for caught mail in tugboat ui for this PR. 

UX
1. Check you email in CAG.
   - [x] Verify that it meets new copy.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
